### ISSUE TITLE
no samples

### ISSUE THREAD

**Issue by KeeeeperOfTheLight** [cocktailpeanut/fluxgym#157]
hello guys!

i installed fluxgym from scratch and i got the same error but instead of can't find samples folder error i got no samples error. 

![Fluxgym s](https://github.com/user-attachments/assets/3dcf033b-5c5a-4eae-b8cf-5e3cd918038f)

every time i solve something i get another one so can anyone help me please to just run it without those errors pfffff.

**Comment by KeeeeperOfTheLight** [comment]
thank you for answering me sir i appreciated.
here is everything tell me what is wrong please.

![Fluxgym a](https://github.com/user-attachments/assets/0ca27432-4e9b-4f3b-805f-7ee2ba51f091)
![Fluxgym b](https://github.com/user-attachments/assets/c789a437-5346-4ac0-9bee-feb92e74647b)
![Fluxgym d](https://github.com/user-attachments/assets/b7a7bf81-78df-4c2e-9260-3dc2fce45bc3)
![Fluxgym e](https://github.com/user-attachments/assets/e593cf0d-d09f-4b86-bd28-771387aa8f84)
![Fluxgym s](https://github.com/user-attachments/assets/3b550fa8-f2c6-423a-863b-2443f1fd56c1)

**Comment by KeeeeperOfTheLight** [comment]
> That's not an error. it simply is saying you didn't set the samples setting. Check if the safetensors files and readme files were correctly created. They should have.
> 
> If you want to auto-generate samples while training, set the `sample image every n steps` as well as the `sample image prompts` fields

thank you for answering me sir i appreciated.
here is everything tell me what is wrong please.

![Fluxgym a](https://github.com/user-attachments/assets/74af9449-8028-4601-99ce-ec5fab552cb0)
![Fluxgym b](https://github.com/user-attachments/assets/38ff53b7-571b-4f89-9364-0fc5fbeefdf7)
![Fluxgym d](https://github.com/user-attachments/assets/aaf7b70a-6257-4f1e-b2ac-d93d33cc3bfb)
![Fluxgym e](https://github.com/user-attachments/assets/113ebe60-678a-4a35-a4a4-95192461c64a)
![Fluxgym s](https://github.com/user-attachments/assets/ea4e7446-7b47-4385-8d16-13330874c967)

**Comment by mostfurious** [comment]
Same here... i very quickly get this error too and it does not move any further. Maybe you have found a solution ?

**Comment by KeeeeperOfTheLight** [comment]
> Same here... i very quickly get this error too and it does not move any further. Maybe you have found a solution ?

sorry dude i found nothing, still w8ing for him to talk back.

**Comment by mostfurious** [comment]
Sort of gave up and used pinocio to install fluxgym. It added like additional ~30GB and installed a ton of crap that i never used.. like VS, but now its working.
Github version size ~30GB.
Pinocio 57.4GB

**Comment by KeeeeperOfTheLight** [comment]
> Sort of gave up and used pinocio to install fluxgym. It added like additional ~30GB and installed a ton of crap that i never used.. like VS, but now its working.
> Github version size ~30GB.
> Pinocio 57.4GB

Pinocio downloaded all the 57Gb in the same pinocio folder ?  or some files ara downloaded in c partition ? Cuz I have no space left in c partition

**Comment by mostfurious** [comment]
57GB in pinocio and i think ~3GB alone for vscode in C, plus CUDA toolkit and some other stuff, i think they were to C as well. But pinocio dir that i have in D drive is 57.4GB

**Comment by KeeeeperOfTheLight** [comment]
> 57GB in pinocio and i think ~3GB alone for vscode in C, plus CUDA toolkit and some other stuff, i think they were to C as well. But pinocio dir that i have in D drive is 57.4GB

Thank you bro for telling me. Good luck and have fun. ❤️

**Comment by KeeeeperOfTheLight** [comment]
i used pinokio and it's the same, any help from anyone i'll be much appreciate it, i did the same settings as above.

![Fluxgym Pin](https://github.com/user-attachments/assets/1259a4e2-48cb-46f6-815a-a73a14edd639)

**Comment by obab30** [comment]
I have the same issue saying "no samples" with fresh installation (manually) of FluxGym.

**Comment by KeeeeperOfTheLight** [comment]
> I have the same issue saying "no samples" with fresh installation (manually) of FluxGym.

trust me installing it manually or by pinokio  you'll get the same error i don't think it's working for anybody, this issue only the owner can fix it. so don't bother yourself i tried everything nothing works, when the owner is ready he will fix it i think.

**Comment by dohtishshit** [comment]
yes i too am having this error its incredibly frustrating both manual and pinokio gives same error owner said earlier in this thread that its not an error, but he should look at it again because its 100% an error I played with every setting doesnt matter same error

**Comment by KeeeeperOfTheLight** [comment]
> yes i too am having this error its incredibly frustrating both manual and pinokio gives same error owner said earlier in this thread that its not an error, but he should look at it again because its 100% an error I played with every setting doesnt matter same error

yup, i'm still w8ing for to answer i hope he does.

**Comment by obab30** [comment]
Okay I fixed the issue for me. I ran the training command in my terminal and it stuck saying that there is a problem with the charset in the caption of one file. One image caption contained a non utf8 symbol. I removed it manually from the txt file and now it works as expected.

Maybe this fixes the issue for you too. Try to look for unusual characters (e.g. öäü) in your captions and remove them.

**Comment by KeeeeperOfTheLight** [comment]
> Okay I fixed the issue for me. I ran the training command in my terminal and it stuck saying that there is a problem with the charset in the caption of one file. One image caption contained a non utf8 symbol. I removed it manually from the txt file and now it works as expected.
> 
> Maybe this fixes the issue for you too. Try to look for unusual characters (e.g. öäü) in your captions and remove them.

Good for you, thank you I'll give it a try no problem, but I remember that I tried without any captions and I got the same error 😔.
So I'll give it a try and I'll put another comment to share the issue with the guys.

**Comment by stchebo** [comment]
Hey, everybody. Same error. No samples. Installed through Pinokio. I will be glad if someone can solve this problem. Thanks

**Comment by cherrycake1030** [comment]
I have also encountered such a problem. I tried closing the program and reopening it, or turning on subtitles or not. With the same picture and the same parameters, sometimes it works for me and sometimes it doesn't. I don't know the reason.

**Comment by omceone** [comment]
ну че, кто то разобрался с этой проблемой?

**Comment by Balalikein** [comment]
Fixed?

**Comment by obab30** [comment]
Okay just for clarification: "no samples" is not really an error. When something (this could be anything) goes wrong during training it would not show an error but end the training with "no samples". So what can you do to find your specific error? You Can start Training as usual. Then go to the Terminal and look for the training call. This is usually a long line with lots of parameters. You can copy this line an execute it manually in the terminal. Then you will most likely get your error. This way i found my issue.

**Comment by stchebo** [comment]
Good evening, everyone. Has anyone solved this problem? I came to the conclusion that I do not have enough RAM 16 gb, although the video card RTX 3060. Opened the task manager there you can see where at about 45 seconds of pamty scaled to 100% and everything went down. Maybe I'm wrong, but I think the reason is this.

**Comment by giilour** [comment]
> Did an install on Ubuntu today and I'm getting the "no samples" error. tried all the fixes here and still no good. It does say that "training complete, check the outputs from lora folder" - it creates a folder but only contains a couple of small files.

The only fix I came up with was to install on windows and not Ubuntu. Will go back to it when it gets more support for linux.

**Comment by hustlecoinnet** [comment]
> Good evening, everyone. Has anyone solved this problem? I came to the conclusion that I do not have enough RAM 16 gb, although the video card RTX 3060. Opened the task manager there you can see where at about 45 seconds of pamty scaled to 100% and everything went down. Maybe I'm wrong, but I think the reason is this.

I also had 16Gb of ram and comfyui and automatic 1111 were failing after a while of processing. I upgraded to 64Gb, and my first processing was using 18Gb of ram, I thought 64 was overkill, but using some other high res models it was going up to 38Gb usage. If you motherboard can handle it - I would go up to atleast 64Gb or ram.

**Comment by stchebo** [comment]
Thanks for the reply. Comfyui seems to work, but apparently 16gb is not enough.

**Comment by noizlab** [comment]
same problem

**Comment by dreamhacking** [comment]
ok, I had the same problem. I tried 2 images 1024X1024 without captions (only trigger word as caption) and it worked.

**Comment by stchebo** [comment]
Thank you! I'll have to try it

**Comment by 8Docent8** [comment]
I'm not an expert, but I've also encountered this problem recently. I create a "caption" in AI + I add it myself + translator. And sometimes this error appears. In this case, I edit the text, remove unnecessary or incomprehensible characters. For example, Roman numerals. If this does not help, then when this error occurs, I go to the "terminal" and look at which image the error occurred on and rewrite the "caption" or delete the image with the description altogether. I delete 2-5 of the 35 images. This helps, but you have to copy the "caption" too much, every time and restart Fluxgym. 

 I hope my experience will help someone. Good luck to everyone!

**Comment by mlstudent2** [comment]
This happened to me with a certain image set. Traced it to an error `illegal char in file (not UTF-8)`. Florence captioning was inserting a non-UTF character and the training script was choking on it. I was able to locate it by going through the generated TXT files in VSCode and doing a search for `[^\x00-\x7f]` with regex option on to locate the character and removed it.

**Comment by automayt** [comment]
For me the version of flux1-dev.sft that it downloaded was corrupted. I copied over the version I use in comfyui and it is working great now. I just copied the logger output and threw it into chatgpt and it immediately identified the issue.

**Comment by jim8266666** [comment]
Yeah i am having this same issue too. Tried every avenue but to avail. I dont wanna give up. Man please help!
